### PR TITLE
Feat/manager-components paginated datagrid & notifications

### PR DESCRIPTION
**Pull request by frenautvh**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/manager-components-grid-notifs
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | 
| License          | BSD 3-Clause

## Description

- add paginated-datagrid to manager-components
- add notifications to manger-components
- update language selector in storybook of manager-components

### CODE REVIEW

**Review by kqesar — DISMISSED** [review]

File: packages/manager-components/src/components/paginated-datagrid/paginated-datagrid.component.tsx [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+                    >
+                      <OsdsText
+                        size={ODS_TEXT_SIZE._500}
+                        color={ODS_THEME_COLOR_INTENT.text}
+                        level={ODS_TEXT_LEVEL.body}
+                        hue={ODS_THEME_COLOR_HUE._500}
+                        style={{
+                          '--osds-text-color-specific-hue':
+                            'var(--ods-color-text-500);',
+                        }}
```
[the remark this answers is not in the document]
Comment (frenautvh) [reply packages/manager-components/src/components/paginated-datagrid/paginated-datagrid.component.tsx]: removed

File: packages/manager-components/src/components/paginated-datagrid/paginated-datagrid.component.tsx [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+                      </OsdsText>
+                      {header.column.getIsSorted() && (
+                        <span
+                          className="align-middle inline-block h-4"
+                          style={{
+                            marginTop:
+                              (header.column.getIsSorted() as string) === 'asc'
+                                ? '-0.75rem'
+                                : '-2rem',
+                          }}
```
[the remark this answers is not in the document]
Comment (frenautvh) [reply packages/manager-components/src/components/paginated-datagrid/paginated-datagrid.component.tsx]: updated with tailwind classes

File: packages/manager-components/src/components/paginated-datagrid/paginated-datagrid.component.tsx [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+                        </span>
+                      )}
+                    </div>
+                  )}
+                </th>
+              ))}
+            </tr>
+          ))}
+        </thead>
+        <tbody style={{ borderBottom: '1px solid var(--ods-color-blue-200)' }}>
```
[the remark this answers is not in the document]
Comment (frenautvh) [reply packages/manager-components/src/components/paginated-datagrid/paginated-datagrid.component.tsx]: removed

File: packages/manager-components/src/components/paginated-datagrid/paginated-datagrid.component.tsx [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+              <td className="text-center" colSpan={columns.length}>
+                <DataGridTextCell>
+                  {t('common_pagination_no_results')}
+                </DataGridTextCell>
+              </td>
+            </tr>
+          )}
+        </tbody>
+      </table>
+      <OsdsPagination
```
[the remark this answers is not in the document]
Comment (frenautvh) [reply packages/manager-components/src/components/paginated-datagrid/paginated-datagrid.component.tsx]: updated the component to make pagination optional by omitting "pagination" & "onPaginationChange" props (also updated the storybook to showcase both examples)

File: packages/manager-components/src/components/paginated-datagrid/paginated-datagrid.component.tsx [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+      }),
+    ),
+    data: items,
+    manualPagination: true,
+    pageCount,
+    state: {
+      pagination: { ...pagination },
+      sorting: [sorting],
+    },
+    enableSortingRemoval: false,
```
[the remark this answers is not in the document]
Comment (frenautvh) [reply packages/manager-components/src/components/paginated-datagrid/paginated-datagrid.component.tsx:95]: i just made sorting optinal (by omitting sorting & onSortChange props) and also added "isSortable" attribute on column type so we can disable sorting on specific columns (updated the storybook to showcase an example with & without sorting enabled)

**Review by cbourgois — APPROVED** [review]

**Review by aboungnaseng-ovhcloud — DISMISSED** [review]

**Comment by frenautvh** [comment]
will be re-opened along with pci-voucher uapp PR

**Pull request closed without merging**